Document/0x05i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -308,23 +308,78 @@ Add the following to build.gradle:
 ### Testing Exception Handling
 
 #### Overview
-
--- TODO [Give an overview about the functionality and it's potential weaknesses] --
+Exceptions can often occur when an application gets into a non-normal or erroneous state. Both in Java and C++ Exceptions can be thrown when such state occurs. 
+Testing exception handling is about reassuring that the application will handle the exception and get to a safe state without exposing any sensitive information at both the UI and the ADB logs.
```

Comment:
I would remove ADB and write "log mechanism". As besides ADB logs, exceptions could also be sent as log to a 3rd party when using a library or saved to a logfile in the data directory.